### PR TITLE
 Change getExposedPorts to return port w/ protocol

### LINKED ISSUE #284: UDP exposed ports not launching correctly
For example in my config if I have:

```
        "ExposedPorts": {
            "20000/udp": {}
        },
```
The launched container is given the following flags:

`$ docker run --rm -d -p 20000:20000 <redacted image>`

It should be:

`$ docker run --rm -d -p 20000:20000/udp <redacted image>`

I think it's just a bug with vscode parsing the config and not knowing to correctly add /udp when it's a udp service.

If I'm doing something wrong myself please do let me know.

### PR DESCRIPTION
**Pull request by PrashanthCorp**
Fixes #284

### CODE REVIEW

File: commands/start-container.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export async function startContainer(context?: ImageNode, interactive?: boolean)
         docker.getExposedPorts(imageToStart.Id).then((ports: string[]) => {
             let options = `--rm ${interactive ? '-it' : '-d'}`;
             if (ports.length) {
-                const portMappings = ports.map((port) => `-p ${port}:${port}`);
+                const portMappings = ports.map((port) => `-p ${port.split("/")[0]}:${port}`);
```
Comment (StephenWeatherford) [inline commands/start-container.ts]: Please comment with what you're expecting and the output you're going for.

Comment (StephenWeatherford) [inline commands/start-container.ts]: Are there are testcases?

Comment (PrashanthCorp) [reply commands/start-container.ts]: current output: 
`-p 8080:8080`

output I'm going for: 
`-p 8080:8080/tcp` , `-p 3010:3010/tcp`

Notice the difference in 8080/tcp. Earlier, we stripped the protocol when returning the function (see `getExposedPorts`). 

The earlier command was: 
`docker run --rm -d -p 8080:8080 node-todo:latest`
Now with the change: 
`docker run --rm -d -p 8080:8080/tcp node-todo:latest`

Comment (PrashanthCorp) [reply commands/start-container.ts]: Done.

Comment (StephenWeatherford) [reply commands/start-container.ts]: I wanted code comments. If you need to explain it to me, you need to explain it to code maintainers.

Comment (PrashanthCorp) [reply commands/start-container.ts]: Ah, my bad. Added comment

**Pull request merged**